keps/sig-api-machinery/5000-api-linting-crd-schema-tooling/README.md

```diff
@@ -265,20 +266,62 @@ Currently, it is proposed that:
+| maxitems | Property | Validates compatibility of changes to maxitems constraints on a property. Net new maxitems constraints or a decrease in maxitems value are flagged | Yes | Yes |
+| maxlength | Property | Validates compatibility of changes to maxlength constraints on a property. Net new maxlength constraints or a decrease in maxlength value are flagged | Yes | Yes |
+| maxproperties | Property | Validates compatibility of changes to maxproperties constraints on a property. Net new maxproperties constraints or a decrease in maxproperties value are flagged | Yes | Yes |
+| minimum | Property | Validates compatibility of changes to minimum constraints on a property. Net new minimum constraints or an increase in minimum value are flagged | Yes | Yes |
+| minitems | Property | Validates compatibility of changes to minitems constraints on a property. Net new minitems constraints or an increase in minitems value are flagged | Yes | Yes |
+| minlength | Property | Validates compatibility of changes to minlength constraints on a property. Net new minlength constraints or an increase in minlength value are flagged | Yes | Yes |
+| minproperties | Property | Validates compatibility of changes to minproperties constraints on a property. Net new minproperties constraints or an increase in minproperties value are flagged | Yes | Yes |
+| required | Property | Validates compatibility of changes to required constraints on a property. Adding new required properties are flagged | Yes | Yes | 
+| type | Property | Validates compatibility of changes to type constraints on a property. Changes to property types are flagged | Yes | Yes |
+| scope | CRD | Validates that scope doesn't change for the CRD | Yes | Yes |
+| existingfieldremoval | CRD | Validates that existing fields are not removed from the CRD | Yes | Yes |
+| storedversionremoval | CRD | Validates that stored versions are not removed from the CRD. Only valid for CRDs that are previously present on a cluster | Yes | Yes |
+| format | Property | Validates compatibility of changes to the format of a property. Changes to property formats are flagged | No | Yes |
+| exclusiveMaximum | Property | Validates compatibility of changes to the exclusiveMaximum of a property. Net new exclusiveMaximum constraints or a decrease in exclusiveMaximum are flagged | No | Yes |
+| exclusiveMinimum | Property | Validates compatibility of changes to the exclusiveMinimum of a property. Net new exclusiveMinimum constraints or an increase in exclusiveMinimum are flagged | No | Yes |
+| uniqueItems | Property | Validates compatibility of changes to the uniqueItems of a property. Net new uniqueItems constraints are flagged | No | Yes | 
+| pattern | Property | Validates compatibility of changes to the pattern constraint of a property. Net new pattern constraints and more restrictive validations are flagged | No | Yes |
```

Comment:
How would you determine if a more restrictive pattern is introduced?

Reply:
Not something I've put much thought into, but I do understand that this could be complex and might not be possible. I figure more nuanced discussion on the _exact_ validation logic would take place during implementation of the validation. 

Happy to remove the "more restrictive validations" piece if needed.